benchmarks/transport_montecarlo_numba_formal_integral_p.py

```diff
@@ -81,19 +107,16 @@ def time_explosion(self):
         # time taken for a photon to move 1 cm
         return 1 / c.c.cgs.value
 
+    # Error: Terminating: fork() called from a process already using GNU OpenMP, this is unsafe.
+    @skip_benchmark
     @parameterize({"p": [0.0, 0.5, 1.0], "Test data": TESTDATA})
     def time_calculate_z(self, p, test_data):
         func = formal_integral.calculate_z
-        inv_t = 1.0 / self.time_explosion
-        len(self.formal_integral_geometry(test_data).r_outer)
+        inv_t = 1.0 / self.verysimple_time_explosion
         r_outer = self.formal_integral_geometry(test_data).r_outer
 
         for r in r_outer:
-            actual = func(r, p, inv_t)
-            if p >= r:
-                assert actual == 0
-            else:
-                np.sqrt(r * r - p * p) * formal_integral.C_INV * inv_t
+            func(r, p, inv_t)
 
     @skip_benchmark
     @parameterize({"p": [0, 0.5, 1], "Test data": TESTDATA})
```

Comment:
Small question, how does `"Test data"` know to send its value to `test_data`?

Reply:
it sends data sequence wise. Parameter name is nothing to do with it.